ApollosProject/apollos-apps
# fix(auth): preserve confirm code

## ✏️ Description
- fixes issue where clicking into confirm code input clears previous value in TV auth flow


## 🔬 To Test
1. Go to confirm code in auth flow
2. Enter a value
3. Exit input
4. Enter input focus again
5. Previous value should still be there

## 📸 Screenshots


## ⚠️ To-do Before Merge



## 📝 Checklist
